src/js/app.js

```diff
@@ -165,3 +165,23 @@ jQuery(document).ready(function() {
     return false;
   } );
 });
+    /*
```

Comment:
Actually this block/function needs to be right above line 167.. Line 167  (```});```) is the end of the jQuery block, which this function needs to be within.